community-operators/myvirtualdirectory/myvirtualdirectory-operator.package.yaml

```diff
@@ -0,0 +1,4 @@
+packageName: myvirtualdirectory-operator
```

Comment:
This should match the operator's directory name, ie. `myvirtualdirectory`
Or the directory can be changed to match the packageName here.